libjulia/backends/evm/EVMCodeTransform.cpp

```diff
@@ -60,12 +60,14 @@ CodeTransform::CodeTransform(
 	solAssert(deposit == 0, "Invalid stack height at end of block.");
 }
 
-void CodeTransform::operator()(const FunctionDefinition&)
+void CodeTransform::operator()(const dev::solidity::assembly::FunctionDefinition&)
```

Comment:
I could use some advice on this, then.  `ErrorReporter` must be including `AST.h` so it can have access to `SourceUnit`.  Would you propose moving by `SourceUnit` to its own header?